plugins/poll/spec/lib/search_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+RSpec.describe Search do
+  fab!(:topic) { Fabricate(:topic) }
+  fab!(:topic2) { Fabricate(:topic) }
+  fab!(:regular_post) { Fabricate(:post, topic: topic, raw: <<~RAW) }
+      Somewhere over the rainbow but no poll.
+    RAW
+
+  fab!(:post_with_poll) { Fabricate(:post, topic: topic2, raw: <<~RAW) }
+      Somewhere over the rainbow with a poll.
+      [poll]
+      * Like
+      * Dislike
+      [/poll]
+    RAW
+
+  before do
+    SearchIndexer.enable
+    Jobs.run_immediately!
+
+    SearchIndexer.index(topic2, force: true)
+    SearchIndexer.index(topic, force: true)
+  end
+
+  after do
+    SiteSetting.poll_enabled = true
+    SearchIndexer.disable
+  end
+
+  describe "in:polls" do
+    it "allows filtering search to posts containing polls" do
```

Comment:
```suggestion
  context "when using in:polls" do
    it "filters posts containing polls" do
```

Feel free to ignore but I think it reads better.